Presentation.Web/Controllers/API/DataProcessingRegistrationController.cs

```diff
@@ -592,7 +671,19 @@ private static DataProcessingRegistrationDTO ToDTO(DataProcessingRegistration va
                     .BasisForTransfer
                     .FromNullable()
                     .Select(basisForTransfer => new NamedEntityWithExpirationStatusDTO(basisForTransfer.Id, basisForTransfer.Name, enabledBasisForTransferOptions.Contains(basisForTransfer.Id) == false))
-                    .GetValueOrDefault()
+                    .GetValueOrDefault(),
+                DataResponsible = value
+                    .DataResponsible
+                    .FromNullable()
+                    .Select(responsible => new ValueWithOptionalRemarkDTO<OptionWithDescriptionAndExpirationDTO>() { 
+                        Value = new OptionWithDescriptionAndExpirationDTO(responsible.Id, responsible.Name, enabledDataResponsibleOptions.Contains(responsible.Id) == false, responsible.Description),
+                        Remark = value.DataResponsibleRemark
+                    })
+                    .GetValueOrFallback(new ValueWithOptionalRemarkDTO<OptionWithDescriptionAndExpirationDTO>()
+                    {
+                        Value = null,
+                        Remark = value.DataResponsibleRemark
```

Comment:
du har lidt dobbeltkonfekt her. Det er jo kun "Value" du ønsker at selecte på, så nøjes med det i stedet for det her.. DTOen skal jo sættes uanset om value er null eller ej. Hvis ikke det var tilfældet, så var det jo fordi kildedata var en enkelt klump

Reply:
Og eftersom remark godt kan stå for sig selv så skal de to ikke være en del af samme property men splittes op i hver deres?